R/get_parsing_tables.R

```diff
@@ -98,13 +98,17 @@ getParsingTables = function(tableSuffix) {
 
 
 createParsingTables = function(
-  tableSuffix = NULL, overwrite = FALSE, dbtype = 'postgres', dbname = NULL,
+  tableSuffix = NULL, overwrite = FALSE, dbtype = 'postgres', dbname = NULL, tabNames = c(),
```

Comment:
- Change tabNames to tableNames for consistency.
- use default of NULL, and change the if statement accordingly